browserstack.json

```diff
@@ -4,15 +4,31 @@
 	"test_framework": "qunit",
 	"test_path": [
 		"test/index.html",
+		"test/startError.html",
 		"test/logs.html",
-		"test/amd.html"
+		"test/amd.html",
+		"test/autostart.html"
 	],
 	"browsers": [
 		"chrome_previous",
 		"chrome_current",
 		"firefox_previous",
 		"firefox_current",
+		"opera_previous",
 		"opera_current",
+		"safari_5_1",
+		{
+			"browser": "safari",
+			"browser_version": "6.0"
+		},
+		{
+			"browser": "safari",
+			"browser_version": "7.0"
+		},
+		{
+			"browser": "safari",
+			"browser_version": "7.1"
+		},
 		"safari_previous",
 		"safari_current",
```

Comment:
> From here is there anything else you recommend to improve the list?

I need to know your planned strategy first. :) Do you plan to test on various equivalent (WebKit-wise) releases or only on one line for each WebKit version? If the latter then you have a duplication: 6.2, 7.1 & 8.0 (covered currently by `safari_previous`) are the same.

Also, you didn't address my comment: https://github.com/jquery/qunit/pull/857/files#r38875240.